src/docs/testing/integration-tests/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Integration Testing
+description: Learn how to write integration tests
+---
+
+This page describes how to use the [integration_test][] package to run
+integration tests. Tests written using this package have the following
+properties:
+ 
+- Compatibility with the `flutter drive` command, for running tests on a
+physical device or emulator.
+- The ability to be run on [Firebase Test Lab], enabling automated testing on a
+variety of devices.
+- Compatibility with [flutter_test] APIs, enabling tests to be written in a
+similar style as [widget tests][]
+
+## Overview
+
+If you're developing a Flutter app, you are probably writing the code on a
+desktop computer, called the **host** machine, and running the app on a mobile
+device or browser, called the **target**.
+
+There are three types of tests that Flutter supports. A **unit** test verifies
+the behavior of a method or class. A **widget test** verifies the behavior of
+Flutter widgets without running the app itself. An **integration test** runs the
+full app on the target device and reports the results back to the host machine.
+
+The flutter_driver package runs integration tests written in Dart on a target
+device and reports the results to the host.
+
+The [integration_test][] package lets you run tests directly on the target
+device. This allows you to run integration tests on multiple Android or iOS
+devices using Firebase TestLab. These tests can also be run using
+flutter_driver.
+
+## Project Setup
+
+Add `integration_test`, `flutter_test`, and optionally `flutter_driver` to your
+pubspec.yaml file:
+
+```yaml
+integration_test: ^0.9.0
```

Comment:
This is soon going to change to 

```yaml
integration_test:
  sdk: flutter
```

Not sure if we should add a comment here to that effect, or at least a TODO for ourselves for it.

Reply:
I created an issue to track for now: https://github.com/flutter/website/issues/5060 - is there an SDK issue I can follow?

Reply:
It looks like it's already in the master channel, so I assume we're waiting for it to land in stable?

Reply:
Correct